site/app/templates/admin/LateDays.twig

```diff
@@ -77,4 +77,34 @@
     $("#user_id").autocomplete({
         source: {{ student_full|raw }}
     });
+    flatpickr("#datestamp", {
+            plugins: [ShortcutButtonsPlugin(
+                    {
+                        button: [
+                            {
+                                label: "Now"
+                            }, 
+                            {
+                                label: "End of time"
+                            }
+                        ],
+                        label: "or",
+                        onClick: (index, fp) => {
+                            let date;
+                            switch (index) {
+                                case 0: 
+                                    date = new Date();
+                                    break;
+                                case 1: 
+                                    date = new Date("01/01/9998");
+                                    break;
+                            }
+                            fp.setDate(date, true);
+                        }
+                    }
+                )],
+            allowInput: true,
+            dateFormat: "m-d-Y"
```

Comment:
Please change this to "Y-m-d" so it matches the rest of the date pickers, or change the rest of the date pickers to match this date format